deltaspike/pom.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.cucumber</groupId>
+        <artifactId>cucumber-jvm</artifactId>
+        <version>4.3.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>cucumber-deltaspike</artifactId>
+    <packaging>jar</packaging>
+    <name>Cucumber-JVM: Deltaspike</name>
+
+    <properties>
+        <deltaspike.version>1.9.0</deltaspike.version>
+        <cdi-api.version>2.0.SP1</cdi-api.version>
+        <jaxb-api.version>2.3.1</jaxb-api.version>
+
+        <weld-se.version>2.4.4.Final</weld-se.version>
+        <openejb-core.version>8.0.0-M1</openejb-core.version>
+        <owb.version>2.0.10</owb.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>cucumber-java</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.deltaspike.cdictrl</groupId>
+            <artifactId>deltaspike-cdictrl-api</artifactId>
+            <version>${deltaspike.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.enterprise</groupId>
+            <artifactId>cdi-api</artifactId>
+            <version>${cdi-api.version}</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>cucumber-junit</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
```

Comment:
You can omit the version here and please avoid the use of properties when not needed.

```
        <dependency>
            <groupId>io.cucumber</groupId>
            <artifactId>cucumber-junit</artifactId>
            <scope>test</scope>
        </dependency>
```

Reply:
> Could you add a `README.md` to explain how this module should be used? It doesn't appear to be trivial.

It can be used in the same why you use weld or obenejb module (if i do it right, it should be an drop-in replacement).
Any preference where i should place the readme, in the basedir of the module? And any tips what should i explain? I will try it, but you see my english is not the best.

Reply:
Yes. Base dir of the module should be fine. Don't worry about your English. Others may be able to improve on it.

I general the information included should be sufficient to get some one who's checked out the https://github.com/cucumber/cucumber-java-skeleton to get started.

So listing the required dependencies, the optional dependencies, and including an example of a step definition class that uses injection should be enough. Feel free to add anything else that is note worthy too.